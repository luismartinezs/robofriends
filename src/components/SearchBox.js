import React from "react";

const SearchBox = ({ searchChange }) => {
  return (
    <div className="pa2">
      <label>
        <input
          type="search"
          placeholder="search robots"
          className="pa3 ba b--green bg-lightest-blue"
          onChange={searchChange}
        />
      </label>
    </div>
  );
};

export default SearchBox;
