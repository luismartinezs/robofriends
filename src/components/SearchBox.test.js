import React from "react";
import { shallow } from "enzyme";
import SearchBox from "./SearchBox";

it("expect to render children correctly", () => {
  expect(shallow(<SearchBox />)).toMatchSnapshot();
});
