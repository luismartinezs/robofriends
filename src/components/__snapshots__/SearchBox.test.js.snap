// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render children correctly 1`] = `
<div
  className="pa2"
>
  <input
    aria-label="Search Robots"
    className="pa3 ba b--green bg-lightest-blue"
    placeholder="search robots"
    type="search"
  />
</div>
`;
